### DISCUSSION TITLE
Mute background music when jingle is playing

### DISCUSSION THREAD

**Discussion by PFLI2** [open-goal/jak-project#2880] [Jak 1]
### What feature is your idea related to?

As a new player I can't help but notice that the celebration of picking up the game's main collectable is quite annoyingly undercut by the stage's music continuing on. I propose an option be added to completely mute the main music track during these celebrations.

### Describe the solution you'd like.

A toggle to completely mute all music when a jingle is playing (such as upon collecting a power cell) to avoid the tracks overlapping. It can then fade back in as the player regains control.

**Comment by ZippyZolton** [comment]
I'm fairly certain it does mute, or at least turn off most of the music tracks.

**Comment by Zedb0T** [comment]
It kind of dampers them, the only time it turns them off is if you are in a scene that already has its own unique music built in.

https://www.youtube.com/watch?v=205KdysMFtk

**Comment by ManDude** [comment]
Yeah there's a separate setting for music volume during movies and it is set to like 30% or something.

**Reply by PFLI2** [reply]
Doesn't that impact all cutscenes? I'm proposing a setting just for these instances.

**Reply by Zedb0T** [reply]
This seems relatively invasive to the original gameplay with little benefit. I think having this as a mod or fork could be nice for the people that want it and implementing it there would not be hard. Ideally in the long term something like this could be packaged up as a tiny file and dropped into a "mods" folder and turned on and off at will.

If you're interested in having it as a mod let me know and I'll make it when I get back in like a week.

**Reply by LuminarLight** [reply]
It could also be a toggleable setting maybe?

**Reply by PFLI2** [reply]
> This seems relatively invasive to the original gameplay with little benefit. I think having this as a mod or fork could be nice for the people that want it and implementing it there would not be hard. Ideally in the long term something like this could be packaged up as a tiny file and dropped into a "mods" folder and turned on and off at will.

It's not my project I don't want to be insistent. I'm just someone who tried the game out and noticed with modern sound hardware the level music is entirely too audible during celebrations with jingles and it bothered me to hear both at once and cut into the satisfaction of it. If the consensus among those who do the work and actually contribute to the project think my idea is bad I respect that. You say it's to little benefit, I think it would be a small change that would go a long way for me. I have a bit of a sensitivity to 2 pieces of music playing simultaneously. I appreciate the mod offer, if this doesn't work out I'd love to take you up on it.

**Reply by Zedb0T** [reply]
Yeah no I definitely don't think the idea is bad, I kind of come from the other end where I am pretty tolerable of most audio and  really can't tell the difference between low quality tracks and high quality tracks most of the time. I do still have some concerns about introducing some unseen bugs or adding a area for regressions to occur but those are solely my concerns and do not reflect the thoughts or any discussion from the main devs. I do understand that each individual may perceive things differently so sorry if my prior response came off as dismissive or insensitive. 

I'm away from my work station for a week but when I return I am down to take a peak and maybe test out a prototype.

**Reply by PFLI2** [reply]
I didn't think you were dismissive or insensitive. Don't worry about it. Take all the time you need. I appreciate any reply at all. It's not about the quality, it's about the clashing songs.
